src/lib/Behat/Component/CreateUrlAliasModal.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace Ibexa\AdminUi\Behat\Component;
+
+use Behat\Mink\Session;
+use Ibexa\Behat\Browser\Component\Component;
+use Ibexa\Behat\Browser\Locator\VisibleCSSLocator;
+
+final class CreateUrlAliasModal extends Component
+{
+    private IbexaDropdown $ibexaDropdown;
+
+    public function __construct(Session $session, IbexaDropdown $ibexaDropdown)
+    {
+        parent::__construct($session);
+        $this->ibexaDropdown = $ibexaDropdown;
+    }
+
+    public function verifyIsLoaded(): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('title'))->assert()->textEquals('Create a new URL alias');
```

Comment:
```suggestion
        $this->getHTMLPage()->setTimeout(3)->find($this->getLocator('title'))->assert()->textEquals('Create a new URL alias');
```
Lets add timeout just ot be sure, this component has a slight animation appearing.